src/html/template-parts/sections/get-started/get-started.html

```diff
@@ -0,0 +1,23 @@
+<section class="get-started">
+    <div class="container">
+		<div class="get-started-inner">
```

Comment:
If you're using '-inner' naming here, it have to contain other elements inside. Just thinking logically - inner part of the section, which contains all child elements.